boards/posix/native_posix/doc/index.rst

```diff
@@ -40,6 +40,11 @@ but it has only been tested on Linux.
    This port will **not** work in Windows Subsystem for Linux (WSL) because WSL
    does not support native 32-bit binaries.
 
+.. note::
+   To run the test cases, either via sanitycheck script or manual via
+   west/cmake, you must have a version of libasan installed (in Ubuntu 18.04
+   install the libasan4 package).
```

Comment:
Please add this dependency to the list of packages in the getting started guide. Too many people are not going to read this note.

Reply:
Most users will not use this so this should not be installed for all.

I'm sure we can find a way to give a reasonable error message when it's not installed.

Reply:
@aescolar @SebastianBoe FYI, there is no need to update the `getting started guide` for ubuntu systems as the `gcc` packages already has a dependency on ASAN.